README.adoc

```diff
@@ -1,3 +1,139 @@
-= Exportdoclet
+= ExportDoclet
 
-image:https://travis-ci.org/johncarl81/exportdoclet.svg?branch=master["Build Status", link="https://travis-ci.org/johncarl81/exportdoclet"]
\ No newline at end of file
+image:https://travis-ci.org/johncarl81/exportdoclet.svg?branch=master["Build Status", link="https://travis-ci.org/johncarl81/exportdoclet"] image:http://img.shields.io/badge/license-ASF2-blue.svg["Apache License 2", link="http://www.apache.org/licenses/LICENSE-2.0.txt"]
```

Comment:
Im a fan of the build status, but the license is shown now on the Github tab bar.  Remove?